examples/src/main/java/nl/knaw/dans/lib/dataverse/ExampleBase.java

```diff
@@ -63,4 +65,12 @@ protected static Path getExamplesRoot() {
         throw new IllegalStateException("Could not find examples root from working dir: " + workingDir);
     }
 
+    protected static void warnForCustomMetadataBlocks(DataverseException e) {
+        if (e.getMessage().contains("Validation Failed") && e.getMessage().contains("is required")) {
+            log.warn("The test assumes only the citation block is mandatory.");
+            log.warn("See https://guides.dataverse.org/en/latest/api/native-api.html#define-metadata-blocks-for-a-dataverse-collection");
+            log.warn("or login as admin and edit the 'metadata fields' in the 'general information' of the dataverse");
```

Comment:
Switching off metadata blocks other than citation is not enough. Still getting an exception: _Validation Failed: Audience is required._

<img width="600" alt="image" src="https://github.com/user-attachments/assets/88761d39-2747-4ce9-8c71-64d378841b92" />